docs/pages/components/table/examples/ExSortMultiple.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <b-field grouped group-multiline>
+      <div class="control is-flex">
+          <b-switch v-model="multipleColumnsSwitch">Sort multiple rows</b-switch>
+      </div>
+      <div class="control is-flex">
+          <b-switch v-model="onlyWhenKeyIsDown">Only enable on ALT + CLICK</b-switch>
+      </div>
+      <div class="control is-flex">
+          <span class="button" @click="resetPriority" :disabled="sortingPrioirty.length === 0">Reset sorting</span>
+      </div>
+    </b-field>
+    <b-table 
+    :data="data"
+    @sort="sortPressed"
+    :backend-sorting="true"
+    :sort-multiple="multipleColumnsSwitch"
+    :sort-multiple-data="sortingPrioirty"
+    >
+      <template slot-scope="props">
+        <b-table-column field="first_name" label="First name" sortable>
+            {{ props.row.first_name }}
+        </b-table-column>
+        <b-table-column field="last_name" label="Last name" sortable>
+            {{ props.row.last_name }}
+        </b-table-column>
+        <b-table-column field="team" label="Team" sortable>
+            {{ props.row.team }}
+        </b-table-column>
+      </template>
+    </b-table>
+  </div>
+</template>
+
+<script>
+import _ from 'lodash'
+
+const dataSource = [
+  { 'id': 2, 'first_name': 'Abbie', 'last_name': 'Archer', 'team': 'Team B'},
+  { 'id': 3, 'first_name': 'Abbie', 'last_name': 'Smith', 'team': 'Team A'},
+  { 'id': 4, 'first_name': 'Jones', 'last_name': 'Smith', 'team': 'Team C'},
+  { 'id': 1, 'first_name': 'Abbie', 'last_name': 'Archer', 'team': 'Team A'},
+]
+  export default {
+      data() {
+          return {
+            multipleColumnsSwitch: true,
+            data: [],
+            sortingPrioirty: [],
+            onlyWhenKeyIsDown: false
+          }
+      },
+      methods: {
+        resetPriority(){
+          this.sortingPrioirty = []
+          this.loadAsyncData()
+        },
+        sortPressed(field, order, event) {
```

Comment:
I don't agree with you, the sort logic should to be in table. For example take a look at https://handsontable.com/docs/7.1.1/demo-multicolumn-sorting.html